### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by alexandesigner** [JoeyAndres/meteor-ionic#76]
@JoeyAndres In addition, I would like to begin the contributions documenting what's been done.. my proposal is to follow the model that the Meteor has been using in your projects (guide, apollostack) to document and be based on Ionic documentation to display the components.. I wonder if you could create a branch for documentation, to encourage PR

**Comment by alexandesigner** [comment]
I suggest be created the repository `docs` in `meteoric124` organization. To encourage PR

**Comment by JoeyAndres** [comment]
Yeah sure I would love another contributor. Currently though, I extract documentation from JavaScript and extract them to a meteor file. I'll need to read into that stuff you are talking about (apollostack).

**Comment by alexandesigner** [comment]
> Currently though, I extract documentation from JavaScript and extract them to the meteor file.

How can I access it? So I can better understand. You will create the repository of documentation in organization?

>  I'll need to read into that stuff you are talking about (apollostack).

No need, I just showed you an example. They use the Hexo (hexo-meteor-theme) to document with markdown of github, is more simple edit and encourage contributions.

**Comment by JoeyAndres** [comment]
@alexandesigner It is generated by gulp. I'll try and document it. It uses dgeni, used by the original ionic, also very new, and barely documented. Eitherway, it might be a good idea to have a new repo. Atm, I generate it in every new version.
